CMakeLists.txt.in

```diff
@@ -0,0 +1,24 @@
+cmake_minimum_required(VERSION 3.0)
+
+string(REPLACE " -Werror" "" TMP_C_DEBUG "${CMAKE_C_FLAGS_DEBUG}")
```

Comment:
Agree w/ @dvrogozh The second thing - u'd better use [target_compile_definitions (PRIVATE)] instead of touching flags directly. BTW, gmock/gtest is a pure C++ project, u don't need to touch [CMAKE_C_FLAGS]

Reply:
What I was trying to achieve exactly was to try and fix build-api-next-defconfig which somehow builds gtest with warnings being treated as errors. This approach did not work, will clarify the reason for this with @azhogov